src/use-input.ts

```diff
@@ -42,7 +42,8 @@ export const useInput = <T extends BaseInput>(host: T) => {
 			const onMouseDown = <T extends Event>(e: T) => {
 				if (
 					e.defaultPrevented ||
-					(e.target as HTMLElement).matches('input, textarea, label')
+					(e.target as HTMLElement).matches('input, textarea, label') ||
+					(e.target as HTMLElement).disabled
```

Comment:
"Property 'disabled' does not exist on type 'HTMLElement'."
You need to cast the element. See https://stackoverflow.com/questions/42803221/disabled-false-does-not-work-in-typescript-2-1
or this page had an better explanation:
https://bobbyhadz.com/blog/typescript-property-disabled-not-exist-type-htmlelement

Reply:
@JockeCK @megheaiulian 
Instead of 'disabled' can I use that kind of condition as below?

```
            if (e.defaultPrevented ||
                e.target.matches('input, textarea, label') ||
                e.target?.classList?.value?.includes("chip-text")) {
                return;
            }
```
Or rather it will not be right, because the element no matter if it would be disabled or not, will always returns true for 'chip-text' check in the if statement and it impact the behavior?